src/constants.rs

```diff
@@ -1,6 +1,6 @@
 //! Constants, most copied from [the game constants].
 //!
-//! Last updated on 2020-06-06, `3bc7337807d380ffb29b9dcec40fc19e707c2476` from
```

Comment:
The old commit id here is is quite old (https://github.com/screeps/common/commit/3bc7337807d380ffb29b9dcec40fc19e707c2476 was back in 2019) while the timestamp is newer, however it seems that all changes done since that commit have made their way into this library.